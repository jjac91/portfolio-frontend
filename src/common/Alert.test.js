import React from "react";
import { render, } from "@testing-library/react";
import Alert from "./Alert";

it("renders without crashing", function() {
  render(<Alert />);
});

it("renders the appropriate message", function() {
    const { asFragment } = render(<Alert type="danger" messages={["hello","goodbye"]} />);
    expect(asFragment()).toMatchSnapshot();
})
