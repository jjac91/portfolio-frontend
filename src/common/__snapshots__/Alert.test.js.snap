// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the appropriate message 1`] = `
<DocumentFragment>
  <div
    class="alert alert-danger"
    role="alert"
  >
    <p
      class="mb-0 small"
    >
      hello
    </p>
    <p
      class="mb-0 small"
    >
      goodbye
    </p>
  </div>
</DocumentFragment>
`;
